plugins/kubernetes-backend/src/types/types.ts

```diff
@@ -228,3 +48,6 @@ export interface KubernetesObjectsProviderOptions {
  * @public
  */
 export type ObjectsByEntityRequest = KubernetesRequestBody;
+
+// TODO remove this re-export as a breaking change after a couple of releases
+export type * from '@backstage/plugin-kubernetes-node';
```

Comment:
There's some prior art for us deprecating these exports too from the main package telling people to import from the new `@backstage/plugin-kubernetes-node` package instead.

It's a little bit clunky as you can't do this: 

```ts
// @public @deprecated
export type * from '@backstage/plugin-kubernetes-node'
```

You'd have to do something like this for each export:

```ts
import * as k8sTypes from '@backstage/plugin-kubernetes-node'

// @public @deprecated
export type ClusterDetails = k8sTypes.ClusterDetails
```

But I think it might be worth it.

Reply:
Hello @benjdlambert  . There is an weird error with api-extractor. It keeps saying that I need to export interfaces that I already exported. For any reason api-extractor is creating some aliases for some imports but then it says that I need to export that aliases also. 
<img width="1102" alt="image" src="https://github.com/backstage/backstage/assets/13066717/707550c7-05e3-4134-b69b-277d039d48af">
<img width="776" alt="image" src="https://github.com/backstage/backstage/assets/13066717/8db7eb46-51b0-4948-bc67-17b1d098e20e">

Do you recognize this type of errors ?